src/applications/personalization/profile360/containers/VAProfileApp.jsx

```diff
@@ -55,7 +63,7 @@ class VAProfileApp extends React.Component {
 
 const mapStateToProps = (state) => {
   return {
-    account: state.user,
+    user: state.user,
     profile: state.vaProfile,
```

Comment:
I know the original reason for this, but I went ahead and corrected this because this is too tricky.